sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/UnitTestResultsImportSensorTest.java

```diff
@@ -152,4 +157,22 @@ public void describe_is_global_and_only_on_language() {
     assertThat(descriptor.languages()).containsOnly("cs");
   }
 
+  @Test
+  public void import_two_reports_for_same_project_should_not_throw() throws Exception {
+    UnitTestResultsAggregator unitTestResultsAggregator = mock(UnitTestResultsAggregator.class);
+    SensorContextTester context = SensorContextTester.create(temp.newFolder());
+    UnitTestResults results = mock(UnitTestResults.class);
+
+    when(unitTestResultsAggregator.aggregate(Mockito.any(WildcardPatternFileProvider.class), Mockito.any(UnitTestResults.class)))
+      .thenReturn(results);
+
+    new UnitTestResultsImportSensor(unitTestResultsAggregator, ProjectDefinition.create(), "cs", "C#")
+      .analyze(context, results);
+
+    new UnitTestResultsImportSensor(unitTestResultsAggregator, ProjectDefinition.create(), "vb", "VB.Net")
```

Comment:
_minor_: the actual language name used by the plugin for vb is `"VB.NET"` in uppercase, although it does not change anything for tests